docker-compose.yml

```
@@ -173,10 +239,15 @@ volumes:
+  frontend-network:
+    driver: bridge
+  backend-network:
+    driver: bridge
+  database-network:
     driver: bridge
-    ipam:
-      config:
-        - subnet: 172.20.0.0/16
\ No newline at end of file
+  monitoring-network:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)